support-workers/src/main/scala/com/gu/zuora/subscriptionBuilders/SubscriptionDataBuilder.scala

```diff
@@ -1,97 +1,8 @@
 package com.gu.zuora.subscriptionBuilders
 
-import cats.data.EitherT
-import cats.implicits._
-import com.gu.support.config.{TouchPointEnvironment, ZuoraContributionConfig}
-import com.gu.support.promotions.{PromoError, PromotionService}
-import com.gu.support.redemption.{InvalidCode, InvalidReaderType}
-import com.gu.support.redemptions.RedemptionData
-import com.gu.support.workers._
-import com.gu.support.workers.states.CreateZuoraSubscriptionState
-import com.gu.support.zuora.api.ReaderType.Gift
-import com.gu.support.zuora.api.{ReaderType, SubscriptionData}
-import com.gu.zuora.subscriptionBuilders.ProductSubscriptionBuilders.buildContributionSubscription
-
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
+import com.gu.support.promotions.PromoError
+import com.gu.support.redemption.InvalidCode
 
 case class BuildSubscribePromoError(cause: PromoError) extends RuntimeException
 
 case class BuildSubscribeRedemptionError(cause: InvalidCode) extends RuntimeException
-
-class SubscriptionDataBuilder(
```

Comment:
Now this class is no longer here the file name should change. Maybe to `BuildSubscribeErrors`?

Reply:
done and I also moved to the handlers package where it's used rather than builders